src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ContestedCaseOrderService.java

```diff
@@ -27,6 +38,44 @@ public void printAndMailGeneralOrderToParties(CaseDetails caseDetails, String au
         }
     }
 
+    public void printAndMailHearingDocuments(CaseDetails caseDetails, String authorisationToken) {
+        if (isContestedPaperApplication(caseDetails)) {
+            Map<String, Object> caseData = caseDetails.getData();
+
+            if (bulkPrintService.shouldPrintForApplicant(caseDetails)) {
+                bulkPrintService.printApplicantDocuments(caseDetails, authorisationToken, createHearingDocumentPack(caseData));
+            }
+
+            bulkPrintService.printRespondentDocuments(caseDetails, authorisationToken, createHearingDocumentPack(caseData));
+            createHearingDocumentPack(caseDetails.getData());
+        }
+    }
+
+    private List<BulkPrintDocument> createHearingDocumentPack(Map<String, Object> caseData) {
+        List<BulkPrintDocument> hearingDocumentPack = new ArrayList<>();
+
+        //LATEST_DRAFT_HEARING_ORDER
+        Map<String, Object> data = (Map) caseData.get(LATEST_DRAFT_HEARING_ORDER);
+        documentHelper.getDocumentLinkAsBulkPrintDocument(data, LATEST_DRAFT_HEARING_ORDER).ifPresent(hearingDocumentPack::add);
+
+        //ADDITIONAL_HEARING_DOCUMENT FROM THE COLLECTION
+        Optional<CaseDocument> latestAdditionalHearingDocument = documentHelper.getLatestAdditionalHearingDocument(caseData);
+        if (latestAdditionalHearingDocument.isPresent()) {
+            String latestAdditionalHearingDocLink = latestAdditionalHearingDocument.get().getDocumentBinaryUrl();
```

Comment:
there's a method called `getCaseDocumentAsBulkPrintDocument` you should use here